### ISSUE TITLE
Dash User Contributed docsets

### ISSUE THREAD

**Issue by Kapeli** [dash-docs-community/dev-docs-requests#1]
Just in case you're not aware of it: https://github.com/Kapeli/Dash-User-Contributions.

**Comment by peterpme** [comment]
Hey @Kapeli, I am! Which is why I started this. Remember when I tweeted / emailed ya?

All the docs I use are out of date. They are hard to keep up to date. I'm not blaming you or the maintainers, but I would like to set up a path moving forward so we _can_ keep things up to date!

Instead of relying on individuals, lets rely on the community. A lot of docs are built using the same static site generators, but everybody's got their own implementation for scraping them.

Let's work together to make keeping unofficial docs up to date and maybe even create some webhooks that automate that process for us!

Just to be clear, this isn't an attack on you by any means. I LOVE Dash, but get upset when I have to fallback to the browser.
